scripts/lib/setup_venv.py

```diff
@@ -60,6 +81,15 @@
     "gifsicle",
 ]
 
+YUM_THUMBOR_VENV_DEPENDENCIES = [
+    "libcurl-devel",  # NOT SURE, NOT TESTED
+    "libjpeg-turbo-devel",  # NOT SURE
```

Comment:
I guess we should test these before merging :)

Reply:
I was able to compile thumbor completely, does this mean it works, or do I have to do some sort of end-to-end testing?